docusaurus.config.js

```diff
@@ -42,14 +42,14 @@ const config = {
           // Please change this to your repo.
           // Remove this to remove the "edit this page" links.
           editUrl:
-            'https://github.com/facebook/docusaurus/tree/main/packages/create-docusaurus/templates/shared/',
+            'https://github.com/xprover/xprover-website/blob/main/',
```

Comment:
'https://github.com/xprover/xprover-website/blob/main/docs/'

Reply:
The correct link is https://github.com/xprover/xprover-website/blob/main/ otherwise it will break the edit button.